### ISSUE TITLE
boolean attributes get non-empty string values repeating the attribute name: attr="attr"

### ISSUE THREAD

**Issue by cetinsert** [tbranyen/diffhtml#242]
After 
```js
innerHTML(target, `<video controls></video>`)
```

expected DOM state:
```html
<video controls></video>
```

got:
```html
<video controls="controls"></video>
```

---

1. Is there a reason why this is the case?
2. Is there a way to get the expected output?
    1. if no, can this be addressed as a bug to fix?
    2. or at the very least handled via a config option?

**Comment by tbranyen** [comment]
This is due to interpolation and how we don't differentiate between the following:

```js
const controls = 'controls';

html`<video ${controls} />`
```

and

```js
const controls = 'controls';

html`<video controls=${controls} />``
```

That said it can and should be fixed. Thanks for opening!

**Comment by tbranyen** [comment]
Opened a fix here: https://github.com/tbranyen/diffhtml/pull/243

**Comment by cetinsert** [comment]
@tbranyen – thank you very much for the immediate response! When is the next release scheduled?

**Comment by tbranyen** [comment]
Just published as 1.0.0-beta.22
